Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -120,7 +61,24 @@ public struct WebView: UIViewRepresentable {
             addObserver()
         }
 
+        public func webView(_ webView: WKWebView, didStartProvisionalNavigation navigation: WKNavigation!) {
+            webView.isHidden = true
+        }
+        
+        public func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: Error) {
+            webView.isHidden = false
```

Comment:
We need to create a task for handling the review failure case.  In which we might add a loading error with the reload button. Thoughts?

Reply:
@rnr ^^^

Reply:
I agree it might be separate ticket